src/app/components/organization-picker.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+
+import { I18nService } from "jslib-common/abstractions/i18n.service";
+import { OrganizationService } from "jslib-common/abstractions/organization.service";
+import { Utils } from "jslib-common/misc/utils";
+import { Organization } from "jslib-common/models/domain/organization";
+
+@Component({
+  selector: "app-organization-picker",
+  templateUrl: "organization-picker.component.html",
+})
+export class OrganizationPickerComponent implements OnInit {
+  constructor(
+    private organizationService: OrganizationService,
+    private i18nService: I18nService,
+    private route: ActivatedRoute
+  ) {}
+
+  activeOrganizationId: string;
+  organizations: Organization[] = [];
+
+  loaded = false;
+
+  get activeOrganization() {
+    return this.organizations.find((org) => org.id == this.activeOrganizationId);
+  }
+
+  async ngOnInit() {
+    this.route.params.subscribe(async (params: any) => {
+      this.activeOrganizationId = params.organizationId;
+      await this.load();
+    });
+  }
```

Comment:
We should avoid using routes unless there are no other option. This is not a "page" component and we can simply use `@Input()` for this which is more testable and easier to work with.

Reply:
Done